ast/compile_test.go

```diff
@@ -2888,6 +2932,11 @@ func TestQueryCompiler(t *testing.T) {
 			imports:  nil,
 			expected: fmt.Errorf("match error\n\tleft  : number\n\tright : null"),
 		},
+		{
+			note:     "undefined function",
+			q:        "data.deadbeef(x)",
+			expected: fmt.Errorf("rego_type_error: undefined function data.deadbeef"),
```

Comment:
[nit] There's no format string here, so this _could_ use `errors.New`. But this seems to be in line with the other cases. Also, I can't think of anymore more minor than this nitpick 😉

Reply:
This is just habit on my part. I'm going to leave as is for now since it's consistent with the rest but will keep errors.New in mind in future.